REQUIRE

```diff
@@ -1,4 +1,5 @@
 julia 0.5
 IntervalSets
+Iterators
```

Comment:
Haha, I'm actually in the middle of making PRs to transition everyone from Iterators to IterTools! Only missed this because it's still in my 0.5 directory.